core/src/main/java/org/jdbi/v3/core/collector/BuiltInCollectorFactories.java

```diff
@@ -30,6 +30,14 @@
 
 /**
  * CollectorFactories for built in JDK and jdbi types.
+ * <p>Supported Collection types:</p>
+ * <ul>
+ *     <li>java.util.Optional&lt;T&gt; (throws an exception if more than one row in result)</li>
+ *     <li>java.util.List&lt;T&gt;</li>
+ *     <li>java.util.Set&lt;T&gt;</li>
+ *     <li>java.util.SortedSet&lt;T&gt;</li>
+ *     <li>java.util.Map&lt;K, V&gt; - for row type Map.Entry&lt;K, V&gt;</li>
```

Comment:
Oops, lots more types supported in MapCollectorFactory that I forgot to add here